db/migrate/20221018144521_add_legacy_flag_to_invoices.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddLegacyFlagToInvoices < ActiveRecord::Migration[7.0]
+  def up
+    change_table :invoices do |t|
+      t.boolean :legacy, null: false, default: false
+      t.float :vat_rate
+    end
+
+    execute "UPDATE invoices SET legacy = 'true';"
+    MigrationTaskJob.set(wait: 40.seconds).perform_later('invoices:fill_vat_rate')
+  end
+
+  def down
+    change_table :invoices do |t|
```

Comment:
Rails/BulkChangeTable: You can combine alter queries using `bulk: true` options.